### ISSUE TITLE
inttypes.h don't have on Visual Studio < 2013

### ISSUE THREAD

**Issue by alagoutte** [tatsuhiro-t/nghttp2#237]
Hi @tatsuhiro-t 

We have a problem (https://code.wireshark.org/review/#/c/8477/ & https://code.wireshark.org/review/#/c/8548/ ) with try to build nghttp 1.0.0 with "old" Visual Studio release (< 2013)
the inttypes.h is don't available...

Change coming from 53bfc70c9e9da8a572bf02a2ac57904548f0ae4a 

May be curl have the same issue @bagder ?

**Comment by tatsuhiro-t** [comment]
So, we found the exception of autoconf manual:
```
but we don't know of any implementation that has stdint.h but not
inttypes.h.
```
Perhaps, we should add compiler specific macro to switch between inttypes.h and stdint.h?

**Comment by tatsuhiro-t** [comment]
Ah, right, that's not C99 compliant.  What is the compiler specific macro for VS < 2013?

**Comment by bagder** [comment]
See https://sourceforge.net/p/predef/wiki/Compilers/

_MSC_VER < 1800 it looks like

**Comment by tatsuhiro-t** [comment]
Thank you!

**Comment by tatsuhiro-t** [comment]
Proposed fix committed 1ddff5b

**Comment by alagoutte** [comment]
Thanks Guy for quick fix 
review in progress for us ( https://code.wireshark.org/review/#/c/8568/ )
